src/falco/script.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eo pipefail
+
+[[ "$par_nogroup" == "false" ]] && unset par_nogroup
+[[ "$par_subsample" == "false" ]] && unset par_subsample
+[[ "$par_bisulfite" == "false" ]] && unset par_bisulfite
+[[ "$par_reverse_compliment" == "false" ]] && unset par_reverse_compliment
+
+IFS=";" read -ra input <<< $par_input
+
+$(which falco) \
```

Comment:
Is there a reason not to just use `falco` here? `which` implied that the binary is in `PATH` anyway?

Reply:
If the Viash command and the exectuable filename are the same, an infinite loop occurs. This approach avoids this loop.

Reply:
Interesting! I have never encountered this behavior before...